### DISCUSSION TITLE
http://tabscifi.com/ is down and unable to read documentation there

### DISCUSSION THREAD

**Discussion by socratical** [tableau/TabPy#501] [Q&A]
There is a link in the readme that takes the reader to the http://tabscifi.com/ website. It is currently down. Any updates on this?

**Comment by 0golovatyi** [comment]
@socratical I was the original creator of the blog. And since I left Tableau to pursue a new opportunity the domain was transferred to the Tableau Advanced Analytics team. I believe they are setting up a new instance for the blog (all posts and comments should be preserved).

Meanwhile, you can use a backup copy of the blog at http://tabscifi.golovatyi.info/. Some links may not work though (you will need to replace tabscifi.com URLs with tabscifi.golovatyi.info in your browser address line when that happens).
